py/thermostat/plot_iir_frequency_response.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+"""
+Small tool to show the frequency response of biquad IIR filters.
```

Comment:
Just use the one in stabilizer.
If you copy things, you have to carry the burden of maintaining that code, checking divergence, porting changes/fixes/improvements in both ways. It's not a good idea if you can just use existing code.

Reply:
And obviously nobody will review duplicate code.